integration/browserify/package.json

```diff
@@ -10,14 +10,14 @@
     "firebase": "^4.3.3"
   },
   "devDependencies": {
-    "browserify": "^14.4.0",
+    "browserify": "^14.5.0",
     "chai": "^4.1.1",
     "karma": "^1.7.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^1.3.0",
     "karma-sauce-launcher": "^1.2.0",
     "karma-spec-reporter": "0.0.31",
     "mkdirp": "^0.5.1",
-    "mocha": "^3.5.0"
+    "mocha": "^4.0.1"
```

Comment:
mocha 4 introduced a breaking change in that it will no longer [force the test process to exit](https://boneskull.com/mocha-v4-nears-release/#mochawontforceexit). Should we also configure our test runner to exit by passing `--exit`?

Reply:
Ahhh thank you sir! I was pulling down Travis to debug this very issue. I will refactor to pass `--exit` see if that alleviates the CI Issues.